src/databricks/labs/ucx/installer/logs.py

```diff
@@ -170,10 +171,15 @@ def snapshot(self) -> list[LogRecord]:
             mode="append",
         )
 
-        error_log_records = [
-            log_record for log_record in log_records if logging.getLevelName(log_record.level) >= logging.ERROR
-        ]
-        if len(error_log_records) > 0:
-            error_log_messages = "\n".join([str(log_record) for log_record in error_log_records])
-            raise RuntimeError(f"Found error log records:\n{error_log_messages}")
+        error_messages = []
+        for log_record in log_records:
+            if logging.getLevelName(log_record.level) < logging.ERROR:
+                continue
+            message = log_record.message
+            # Ignore the prompt to troubleshoot, which is the error
+            if 'databricks workspace export /' in message:
```

Comment:
Where is this line coming from?

Reply:
From task runner, prompting to fetch logs directly